packages/evm/src/eof/errors.ts

```diff
@@ -3,157 +3,157 @@ import { EthereumJSErrorWithoutCode } from '@ethereumjs/util'
+  RJUMPV_TABLE_SIZE0: 'invalid RJUMPV: table size 0',
+  UNREACHABLE_CODE_SECTIONS: 'unreachable code sections',
+  UNREACHABLE_CODE: 'unreachable code (by forward jumps)',
+  DATALOADN_OOB: 'DATALOADN reading out of bounds',
+  MAX_STACK_HEIGHT_VIOLATION: 'Max stack height does not match the reported max stack height',
+  STACK_UNDERFLOW: 'Stack underflow',
+  STACK_OVERFLOW: 'Stack overflow',
+  UNSTABLE_STACK: 'Unstable stack (can reach stack under/overflow by jumps)',
+  RETF_NO_RETURN: 'Trying to return to undefined function',
+  RETURN_STACK_OVERFLOW: 'Return stack overflow',
+  INVALID_EXTCALL_TARGET: 'invalid extcall target: address > 20 bytes',
+  INVALID_RETURN_CONTRACT_DATA_SIZE: 'invalid RETURNCONTRACT: data size lower than expected',
+  INVALID_EOF_FORMAT: 'invalid EOF format',
 } as const
 
 export type SimpleErrors = (typeof SimpleErrors)[keyof typeof SimpleErrors]
 
 export const SimpleErrors = {
-  minContainerSize: 'err: container size less than minimum valid size',
-  invalidContainerSize: 'err: invalid container size',
-  typeSize: 'err: type section size invalid',
-  code0msh: 'err: computed max stack height for code section 0 does not match expect',
-  underflow: 'err: stack underflow',
-  code0IO: 'err: input and output of first code section must be 0',
-  VerifyUint: 'Uint does not match expected value ',
-  VerifyBytes: 'Bytes do not match expected value',
-  TypeSize: 'missing type size',
-  InvalidTypeSize: 'err: type section invalid',
-  CodeSize: 'missing code size',
-  CodeSectionSize: 'code section should be at least one byte',
-  InvalidCodeSize: 'code size does not match type size',
-  DataSize: 'missing data size',
-  TypeSections: 'need to have a type section for each code section',
-  Inputs: 'expected inputs',
-  Outputs: 'expected outputs',
-  MaxInputs: 'inputs exceeds 127, the maximum, got: ',
-  MaxOutputs: 'outputs exceeds 127, the maximum, got: ',
-  Code0Inputs: 'first code section should have 0 inputs',
-  Code0Outputs: 'first code section should have 0 outputs',
-  MaxStackHeight: 'expected maxStackHeight',
-  MaxStackHeightLimit: 'stack height limit of 1024 exceeded: ',
-  MinCodeSections: 'should have at least 1 code section',
-  MaxCodeSections: 'can have at most 1024 code sections',
-  CodeSection: 'expected a code section',
-  DataSection: 'Expected data section',
-  DanglingBytes: 'got dangling bytes in body',
+  MIN_CONTAINER_SIZE: 'err: container size less than minimum valid size',
+  INVALID_CONTAINER_SIZE: 'err: invalid container size',
+  TYPE_SIZE: 'err: type section size invalid',
+  CODE0_MSH: 'err: computed max stack height for code section 0 does not match expect',
+  UNDERFLOW: 'err: stack underflow',
+  CODE0_IO: 'err: input and output of first code section must be 0',
+  VERIFY_UINT: 'Uint does not match expected value ',
+  VERIFY_BYTES: 'Bytes do not match expected value',
+  TYPE_SIZE_2: 'missing type size',
```

Comment:
```suggestion
  TYPE_SIZE: 'missing type size',
```
typo?

Reply:
This looks like a typo, but it also seems like this `TYPE_SIZE_2` isn't used anywhere? :thinking:

Reply:
This wasn't a typo, there were previously two keys with the same name but different casing: `TypeSize` and `typeSize`. Since the second one is not used anywhere, I'll just remove it for now if there are no objections.